Code/GeorgiaLibrarySystem-/Core/Core.csproj

```diff
@@ -30,6 +30,15 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'bad test project|AnyCPU'">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\bad test project\</OutputPath>
```

Comment:
whats up with this test project and the rest of the changes in core?

Reply:
idk, i got it when i merged with dev